src/main/java/com/getindata/connectors/http/internal/table/lookup/BodyBasedRequestFactory.java

```diff
@@ -53,9 +53,15 @@ protected Logger getLogger() {
         return log;
     }
 
-    private URI constructGetUri() {
+    URI constructBodyBasedUri(LookupQueryInfo lookupQueryInfo) {
```

Comment:
Small comment, I was confused by the name of this method. I wonder if constructUri or addQueryInfoToURI would be more appropriate.
When we add path parameters I assume there would be another map in LookupQueryInfo that we would use to replace content in the parameterised baseUrl.

Reply:
> When we add path parameters 

This is not the scope of the current PR. It might be revisited with a PR dedicated to path parameters.

Reply:
> > When we add path parameters
> 
> This is not the scope of the current PR. It might be revisited with a PR dedicated to path parameters.

OK